backend/usecases/offerCases.go

```diff
@@ -28,6 +30,23 @@ func (rc *OfferController) GetLastOffers(before time.Time, size int) ([]models.O
 
 // InsertOffer inserts the given offer into the database, calling the DAL package function.
 // It returns error != nil in case some error occured.
-func (rc *OfferController) InsertOffer(offer models.Offer) error {
+func (rc *OfferController) InsertOffer(offer models.Offer, telephone bool, email bool) error {
+	u, err := rc.userDAL.GetUserInfo(offer.Userid)
+
```

Comment:
nit: usually we don't use a space between error assignment and check.

Reply:
Corrected in 7e6c693